pages/index.js

```diff
@@ -30,11 +30,21 @@ const Index = () => {
           <Header title="Home Page" />
           {isLoading && <strong>Loading...</strong>}
           {!isLoading &&
-            data.inventory.records.map(r => (
-              <div key={`product-${r.id}`}>
-                <strong>{r.fields['Product Name'][0]}</strong>
-              </div>
-            ))}
+            data.inventory.records.map(r => {
+              const [productName] = r.fields['Product Name'];
```

Comment:
After going back through this, I think we made a few errors. Since fields is an object, it'll need to be object destructuring not array destructuring, and the field names would need to match the variables we were trying to set. The issue would then be the spacing provided by the airtable fields. To make this work, you would need to rename the 'product name' field in airtable to match the variable with no spacing (e.g. 'productName' vs 'Product Name'.

Which would then look more like 
const { Manufacturer, productName } = r.fields;

Reply:
Nice catch! Yeah, this makes sense. Thanks for the followup JJ!